.travis.yml

```diff
@@ -4,8 +4,6 @@ sudo: false
 
 python:
     - "2.7"
-    - "3.4"
-    - "3.5"
```

Comment:
The latest version of Colorama does support Python 3.5. Are you sure about this?

Reply:
Yes, Colorama support Python 3.5. 

Here is quote from Colorama docs https://github.com/tartley/colorama#dependencies

> Tested on Python 2.7, 3.5, 3.6, 3.7 and 3.8.

Reply:
@renskiy you're right, we should support Python 3.5.

I've made an additional commit and returned Python 3.5 to travis.

Reply:
Oh, but forgot do the same with fabricio doc. Need one more commit :slightly_smiling_face:

Reply:
@renskiy I think you should make squash commits before merging and combine my 7 commits into 1. 

Github interface should support squashing before merging.